migrate/playbooks/rds-timescale-live-migration.md

```diff
@@ -0,0 +1,410 @@
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/navigate-to-ec2-instance.jpg"
+ alt="Navigate to your EC2 instance"/>
+
+2. Select the "SSH client" tab.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/connect-ssh.jpg"
+ alt="Connect to your EC2 using SSH client"/>
+
+3. Connect to your EC2 instance using the "Key pair" you received while creating the EC2 instance.
+```sh
+chmod 400 <key-pair>.pem
+ssh -i "<key-pair>.pem" ubuntu@<EC2 instance's Public IPv4>
+```
+
+4. Install PostgreSQL client and pgcopydb.
+```sh
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+wget -qO- https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo tee /etc/apt/trusted.gpg.d/pgdg.asc &>/dev/null
+sudo apt update
+sudo apt install postgresql-client-15 pgcopydb -y
+psql --version && pg_dump --version && pgcopydb --version
+```
+
+5. To allow an EC2 instance to connect to an RDS instance, you need to modify the
+security group associated with your RDS instance.
+* Note the Private IPv4 address of your EC2 instance.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/note-private-ipv4.jpg"
+ alt="Note the private IPv4 address"/>
+
+* Go to the security group associated with your RDS instance and select "Edit inbound rules".
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/edit-security-group.jpg"
+ alt="Edit the security group"/>
+
+* Click "Add rule". In the "Type" field, select "PostgreSQL". For "Source", select
+"Custom" and input the Private IPv4 address of your EC2 instance. Add a suitable
+description for this rule. Finally, click on "Save rules" to apply the changes.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/add-new-rule.jpg"
+ alt="Add new rule"/>
+
+6. Verify the connection to the RDS service from your EC2 instance. Please note,
+you will be prompted to enter the master password. This should be the same password
+that you used when creating your AWS RDS service.
+```sh
+# psql -h <rds_endpoint> -p <rds_port> -U postgres -c "select 1"
+psql -h aws-rds-low-downtime-migration-db.c8tzn206yp6f.us-west-2.rds.amazonaws.com -p 5432 -U postgres -c "select 1"
+Password for user postgres:
+ ?column?
+----------
+        1
+(1 row)
+```
+
+## Performing data migration
+<GettingHelp />
+
+The migration process consists of the following steps:
+1. Set up a target database instance in Timescale.
+1. Prepare the source database for the live migration.
+1. Set up a replication slot and snapshot.
+1. Migrate roles and schema from source to target.
+1. Follow the remaining steps from "Live migration" documentation.
```

Comment:
We instruct to follow the steps, or the live migration consists of the following steps. However, stating "follow the remaining steps" in the final point is not effective. I suggest changing it to simply "Perform the live migration" as the last point.